geode-wan/src/main/java/org/apache/geode/cache/wan/internal/WanCopyRegionFunctionService.java

```diff
@@ -41,11 +44,18 @@ public class WanCopyRegionFunctionService implements CacheService {
 
   @Override
   public boolean init(Cache cache) {
+    int maxConcurrentThreads = SystemProperty
+        .getProductIntegerProperty(
+            SystemPropertyHelper.WAN_COPY_REGION_MAX_CONCURRENT_THREADS, 10);
+    return init(maxConcurrentThreads);
+  }
+
+  @VisibleForTesting
+  boolean init(int maxConcurrentThreads) {
     String WAN_COPY_REGION_FUNCTION_EXECUTION_PROCESSOR_THREAD_PREFIX =
```

Comment:
Why is this not a constant member variable?

Reply:
Changed